### PR TITLE
[ko] Update outdated files in dev-1.21-ko.3 (p2)

### PR DESCRIPTION
**Pull request by jihoon-seo**
Related issue: #28079

Update outdated files in the upstream/dev-1.21-ko.3 branch.

M1 - M19 are covered in this PR.

> 1. [ ]  M1.  `content/en/community/_index.html`  | 2(+XS) 1(-)
> 2. [ ]  M2.  `content/en/docs/concepts/cluster-administration/addons.md`  | 1(+XS) 1(-)
> 3. [ ]  M3.  `content/en/docs/concepts/configuration/secret.md`  | 2(+XS) 2(-)
> 4. [ ]  M4.  `content/en/docs/concepts/containers/images.md`  | 1(+XS) 1(-)
> 5. [ ]  M5.  `content/en/docs/concepts/extend-kubernetes/operator.md`  | 2(+XS) 0(-)
> 6. [ ]  M6.  `content/en/docs/concepts/overview/what-is-kubernetes.md`  | 1(+XS) 1(-)
> 7. [ ]  M7.  `content/en/docs/concepts/overview/working-with-objects/common-labels.md`  | 3(+XS) 2(-)
> 8. [ ]  M8.  `content/en/docs/concepts/overview/working-with-objects/namespaces.md`  | 1(+XS) 1(-)
> 9. [ ]  M9.  `content/en/docs/concepts/scheduling-eviction/_index.md`  | 32(+M) 3(-)
> 10. [ ]  M10.  `content/en/docs/concepts/scheduling-eviction/kube-scheduler.md`  | 2(+XS) 0(-)
> 11. [ ]  M11.  `content/en/docs/concepts/scheduling-eviction/pod-overhead.md`  | 1(+XS) 1(-)
> 12. [ ]  M12.  `content/en/docs/concepts/scheduling-eviction/resource-bin-packing.md`  | 1(+XS) 1(-)
> 13. [ ]  M13.  `content/en/docs/concepts/scheduling-eviction/scheduler-perf-tuning.md`  | 1(+XS) 1(-)
> 14. [ ]  M14.  `content/en/docs/concepts/storage/persistent-volumes.md`  | 3(+XS) 3(-)
> 15. [ ]  M15.  `content/en/docs/concepts/storage/storage-classes.md`  | 32(+M) 2(-)
> 16. [ ]  M16.  `content/en/docs/concepts/workloads/pods/disruptions.md`  | 4(+XS) 2(-)
> 17. [ ]  M17.  `content/en/docs/concepts/workloads/pods/init-containers.md`  | 1(+XS) 2(-)
> 18. [ ]  M18.  `content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md`  | 1(+XS) 1(-)
> 19. [ ]  M19.  `content/en/docs/reference/command-line-tools-reference/feature-gates.md`  | 10(+S) 4(-)

### CODE REVIEW

**Comment by jihoon-seo** [comment]
나중에 merge 직전에 squash 하겠습니다!
/unhold

File: content/ko/docs/concepts/scheduling-eviction/_index.md [hunk 1]
```diff
@@ -1,7 +1,43 @@
...
+no_list: true
 ---
+
++In Kubernetes, scheduling refers to making sure that {{<glossary_tooltip text="Pods" term_id="pod">}}
++are matched to {{<glossary_tooltip text="Nodes" term_id="node">}} so that the
++{{<glossary_tooltip text="kubelet" term_id="kubelet">}} can run them. Preemption
++is the process of terminating Pods with lower {{<glossary_tooltip text="Priority" term_id="pod-priority">}}
++so that Pods with higher Priority can schedule on Nodes. Eviction is the process
++of terminating one or more Pods on Nodes.
++
```
Comment (pjhwa) [inline content/ko/docs/concepts/scheduling-eviction/_index.md:19]: 번역이 되지 않았습니다.

File: content/ko/docs/concepts/scheduling-eviction/_index.md [hunk 2]
```diff
@@ -1,7 +1,43 @@
...
+쿠버네티스에서, 스케줄링은 {{<glossary_tooltip text="kubelet" term_id="kubelet">}}이 파드를 실행할 수 있도록 
+{{<glossary_tooltip text="파드" term_id="pod">}}를 
+{{<glossary_tooltip text="노드" term_id="node">}}에 할당하는 것을 말한다.
+선점(Preemption)은 {{<glossary_tooltip text="우선순위" term_id="pod-priority">}}가 높은 파드가 노드에 스케줄될 수 있도록
```
Comment (pjhwa) [inline content/ko/docs/concepts/scheduling-eviction/_index.md:23]:
```suggestion
선점은 {{<glossary_tooltip text="우선순위" term_id="pod-priority">}}가 높은 파드가 노드에 스케줄될 수 있도록 
```

- 앞서 한영 병기하였으므로, 추가로 할 필요는 없을 것 같습니다.

File: content/ko/docs/concepts/scheduling-eviction/_index.md [hunk 3]
```diff
@@ -1,7 +1,43 @@
...
+* [스케줄링 프레임워크](/docs/concepts/scheduling-eviction/scheduling-framework/)
+* [스케줄러 성능 튜닝](/ko/docs/concepts/scheduling-eviction/scheduler-perf-tuning/)
+* [확장된 리소스를 위한 리소스 빈 패킹(bin packing)](/ko/docs/concepts/scheduling-eviction/resource-bin-packing/)
+
+## 파드 중단(disruption)
+
+{{<glossary_definition term_id="pod-disruption" length="all">}}
+
+* [파드 우선순위와 선점(Preemption)](/docs/concepts/scheduling-eviction/pod-priority-preemption/)
+* [노드-압박 축출(Eviction)](/docs/concepts/scheduling-eviction/node-pressure-eviction/)
```
Comment (pjhwa) [inline content/ko/docs/concepts/scheduling-eviction/_index.md:42]:
```suggestion
* [파드 우선순위와 선점](/docs/concepts/scheduling-eviction/pod-priority-preemption/)
* [노드-압박 축출](/docs/concepts/scheduling-eviction/node-pressure-eviction/)
```

File: content/ko/docs/concepts/workloads/pods/disruptions.md [hunk 4]
```diff
@@ -79,13 +79,15 @@ weight: 60
   ([다중 영역 클러스터](/docs/setup/multiple-zones)를 이용한다면)에
   애플리케이션을 분산해야 한다.
 
-자발적 중단의 빈도는 다양하다. 기본적인 쿠버네티스 클러스터에서는 자발적인 운영 중단이 전혀 없다.
+자발적 중단의 빈도는 다양하다. 기본적인 쿠버네티스 클러스터에서는 자동화된 자발적 중단은 발생하지 않는다 (사용자가 지시한 자발적 중단만 발생한다).
```
Comment (pjhwa) [inline content/ko/docs/concepts/workloads/pods/disruptions.md:82]:
```suggestion
자발적 중단의 빈도는 다양하다. 기본적인 쿠버네티스 클러스터에서는 자동화된 자발적 중단은 발생하지 않는다(사용자가 지시한 자발적 중단만 발생한다).
```

- 괄호 앞의 공백은 없는 것이 좋겠습니다.

File: content/ko/docs/concepts/workloads/pods/disruptions.md [hunk 5]
```diff
@@ -79,13 +79,15 @@ weight: 60
...
 그러나 클러스터 관리자 또는 호스팅 공급자가 자발적 중단이 발생할 수 있는 일부 부가 서비스를 운영할 수 있다.
 예를 들어 노드 소프트웨어의 업데이트를 출시하는 경우 자발적 중단이 발생할 수 있다.
 또한 클러스터(노드) 오토스케일링의 일부 구현에서는
 단편화를 제거하고 노드의 효율을 높이는 과정에서 자발적 중단을 야기할 수 있다.
 클러스터 관리자 또는 호스팅 공급자는
 예측 가능한 자발적 중단 수준에 대해 문서화해야 한다.
+파드 스펙 안에 [프라이어리티클래스 사용하기](/ko/docs/concepts/configuration/pod-priority-preemption/)와 같은 특정 환경설정 옵션
+또한 자발적 (+ 비자발적) 중단을 유발할 수 있다.
```
Comment (pjhwa) [inline content/ko/docs/concepts/workloads/pods/disruptions.md:90]:
```suggestion
또한 자발적(+ 비자발적) 중단을 유발할 수 있다.
```

**Review by pjhwa — COMMENTED** [review]
@jihoon-seo 기여 감사합니다!!! 👍🏼 

작은 의견 몇 개 드립니다. 확인 부탁드려요~

File: content/ko/docs/concepts/scheduling-eviction/_index.md (hunk 1 above)
Comment (jihoon-seo) [reply content/ko/docs/concepts/scheduling-eviction/_index.md:19]: 영문 원문을 붙여넣기 해 놓고 번역 작업을 했는데, 영문 원문을 지우는 것을 깜빡했습니다.
지우겠습니다. :)

**Comment by jihoon-seo** [comment]
@pjhwa 리뷰 감사합니다.
주신 의견을 모두 반영하고 squash 했습니다.
검토 부탁드립니다! ^^

**Comment by jihoon-seo** [comment]
> 1. [ ] R1. `content/en/docs/concepts/configuration/pod-priority-preemption.md` -> `content/en/docs/concepts/scheduling-eviction/pod-priority-preemption.md`

도 이 PR에 반영했습니다!
/unhold

**Comment by pjhwa** [comment]
수고 많으셨습니다!

/lgtm

**Comment by seokho-son** [comment]
감사합니다~!

/approve

**Pull request merged**